gitops/components/messaging/rabbitmq/cluster/README.md

```diff
@@ -0,0 +1,3 @@
+## Cluster RabbitMQ deployment on openshift
+
+As we cannot create role and role binding now, we cannot do RabbitMQ clustering.
```

Comment:
Chris and Adam can create role bindings, just not the rest of the team.

Reply:
change the wording